src/gluonts/dataset/repository/datasets.py

```diff
@@ -3,7 +3,8 @@
 from functools import partial
 from pathlib import Path
 
-from gluonts.dataset.common import load_datasets, TrainDatasets
+from gluonts.dataset.common import TrainDatasets, load_datasets
+from gluonts.dataset.repository.lstnet import generate_lstnet_dataset
```

Comment:
I am not sure about the name `lstnet` since the actual datasets do not have to do anything with lstnet, right? I guess they are just public datasets? The lstnet part could be just in a descriptive comment... 

Not a strong opinion... just trying to figure out a way to make this more intuitive since the lstnet name is not associated with datasets (like m4) so it is not straightforward for an external user.

Reply:
This is rather private stuff as the user interface should only be 'datasets.py', I have renamed the file with '_' to make it clearer.

For Lstnet, I would rather keep it as the script reuse the lstnet processed versions (UCI are unprocessed and different).